docs/pages/textarea.js

```diff
@@ -11,7 +9,26 @@ import docgen, { type DocGen } from '../components/docgen.js';
 export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen |}): Node {
   return (
     <Page title="TextArea">
```

Comment:
super nit: I've been using `{generatedDocGen?.displayName}` for the Page `title` and PageHeader `name`. It doesn't matter, but it's kinda neat.